### DISCUSSION TITLE
Delete custom icon shape

### DISCUSSION THREAD

**Discussion by I21b** [LawnchairLauncher/lawnchair#4387] [Q&A] [answered]
How to delete an unused custom icon shape?
It's annoying because when I'm not using it, it's still there.

**Comment by tgex0** [comment, marked as answer]
Unfortunately, this does not seem to be possible. Once you create a custom icon shape, it hangs around (unless the app's storage is cleared).
If you like, you could perhaps create a 'Feature request' in the Issues section: https://github.com/LawnchairLauncher/lawnchair/issues

I'd like this option, too. I like to be able to reverse any changes I make to the UI.

Sorry for the time gap between your post and mine. I'm new to the project and just having a look around.
